test/functional/ex_cmds/oldfiles_spec.lua

```diff
@@ -1,49 +1,92 @@
-local h = require('test.functional.helpers')
+local Helpers = require('test.functional.helpers')
+local Screen  = require('test.functional.ui.screen')
 
-local buf     = h.curbufmeths
-local command = h.command
-local eq      = h.eq
-local execute = h.execute
-local feed    = h.feed
-local nvim    = h.nvim
+local buf         = Helpers.curbufmeths
+local command     = Helpers.command
+local eq          = Helpers.eq
+local execute     = Helpers.execute
+local feed        = Helpers.feed
+local nvim        = Helpers.nvim
+local nvim_prog   = Helpers.nvim_prog
+local set_session = Helpers.set_session
+local spawn       = Helpers.spawn
```

Comment:
Though we haven't come up with a strong set of best practices for lua, I suggest following the patterns from other test files in the meantime, even if they aren't the best. Example: https://github.com/mhinz/neovim/blob/tests/improve-oldfiles/test/functional/ex_cmds/menu_spec.lua#L2-L4